src/main/java/ch/njol/skript/registrations/Feature.java

```diff
@@ -29,6 +29,7 @@
  * Experimental feature toggles as provided by Skript itself.
  */
 public enum Feature implements Experiment {
+	FOR_EACH_LOOPS("for loop", LifeCycle.EXPERIMENTAL, "for [each] [loop[s]]")
```

Comment:
I don't think `using experiment for` is great. I would either require `each` or do something like `for (each [loop[s]]|loop[s])`

Reply:
for [each] loop[s] would be my preference